runtime/server.go

```diff
@@ -0,0 +1,56 @@
+package runtime
```

Comment:
Lets go with interfaces only for now. Server we dont need introduce now.
Later on after POC we will write swagger spec for both gateway and runtime and generate code automatically